returnn/hdf.py

```diff
@@ -346,6 +364,276 @@ def run(self):
+        :param rounding: defines how timestamps should be rounded if they do not exactly fall onto a sample:
+            start_and_duration will round down the start time and the duration of the segment
+            rasr_compatible will round up the start time and round down the end time
+        :param round_factor: do the rounding based on a sampling rate that is scaled down by this factor
+        :param target_sampling_rate: desired sampling rate for the HDF, data will be resampled to this rate if needed
+        :param concurrent: Split up the list of splits into this many concurrent jobs.
+            Recommended is about one unit of concurrency per 1000h of audio.
+            This value affects how I/O efficient the job is. With increasing concurrency
+            the I/O efficiency decreases as recordings may end up having to be read multiple
+            times from disk.
+            Within job concurrency is handled by the multiprocessing library, using `num_workers`
+            as parallelism factor.
+            Note that within-job concurrency is more I/O efficient than between-job concurrency,
+            so prefer increasing `num_workers` over increasing `concurrent`, when possible.
+        :param num_workers: Num subprocs (multiprocessing.Pool size) used for parallel recording processing.
+            It can be increased to e.g. match the number of CPU cores on a big cluster machine,
+            and the job will stay I/O efficient.
+        """
+        self.bliss_corpus = bliss_corpus
+        self.splits = splits
+        if ffmpeg_output_args is not None:
+            assert ffmpeg_output_args, "ffmpeg_output_args must not be empty"
+            assert output_dtype is None, (
+                "when using ffmpeg_output_args, output_dtype must be None as the data is always written as raw bytes."
+            )
+            self.output_dtype = "int16"
+        else:
+            assert output_dtype in ["float64", "float32", "int16"]
+            self.output_dtype = output_dtype
+        self.ffmpeg_output_args = ffmpeg_output_args
+        self.multi_channel_strategy = multi_channel_strategy or BlissToAudioHDFJob.Mixdown()
+        assert isinstance(self.multi_channel_strategy, (BlissToAudioHDFJob.Mixdown, BlissToPcmHDFJob.PickNth)), (
+            f"{multi_channel_strategy} is not a supported multi-channel strategy."
+        )
+        self.rounding = rounding
+        assert round_factor > 0
+        self.round_factor = round_factor
+        self.returnn_root = returnn_root
+        self.target_sampling_rate = target_sampling_rate
+
+        assert concurrent > 0
+        self.concurrent = concurrent
+        assert num_workers > 0
+        self.num_workers = num_workers
+
+        self.out_hdfs = [self.output_path(f"{i + 1:0d}.hdf") for i in range(len(splits))]
+
+        self.rqmt = {
+            "cpu": 1 + (num_workers // 3),  # 3 workers per CPU core + 1 for the main process
```

Comment:
Before, you had:
```suggestion
            "cpu": 1 + (num_workers // 2),  # 2 workers per CPU core + 1 for the main process
```
Why now `//3` instead?

Reply:
Observing the job I noticed //3 reflects the actual CPU usage better than //2.